### ISSUE TITLE
a11y module shows disabled point markers 

### ISSUE THREAD

**Issue by KacperMadej** [highcharts/highcharts#15480]
#### Behaviour
accessibility module is showing disabled markers after chart resize or update.
Without the module markers are working correctly - still hidden after resize or update if were disabled at the start.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/msdr1ohx/3/

#### Product version
9.0.1

**Comment by KacperMadej** [comment]
Workaround: use small markers - https://jsfiddle.net/BlackLabel/msdr1ohx/4/
